vars/skipStage.groovy

```diff
@@ -217,6 +229,7 @@ boolean call(Map config = [:]) {
     }
 
     String target_branch = env.CHANGE_TARGET ? env.CHANGE_TARGET : env.BRANCH_NAME
+    String hw_distro = hwDistroTarget2().join("")
```

Comment:
I think this is what [hwDistroTarget()](https://github.com/daos-stack/pipeline-lib/blob/668ba308eadd4cc9980cf758190c752e1e073546/vars/hwDistroTarget.groovy) does/is for.

But again, I'm not sure I see the need for this to be an argument to `skip_ftest_hw()` and not just determined internally in `skip_ftest_hw()`.

Reply:
Thanks. Agreed.